helm-charts/binderhub/values.yaml

```diff
@@ -11,31 +17,44 @@ binderhub:
       nginx.ingress.kubernetes.io/proxy-body-size: 256m
       kubernetes.io/ingress.class: nginx
       cert-manager.io/cluster-issuer: letsencrypt-prod
+  initContainers:
+    - name: git-clone-templates
+      image: alpine/git
+      args:
+        - clone
+        - --single-branch
+        - --branch=master
+        - --depth=1
+        - --
+        - https://github.com/pangeo-data/pangeo-custom-binderhub-templates.git
+        - /etc/binderhub/custom
+      securityContext:
+        runAsUser: 0
+      volumeMounts:
+        - name: custom-templates
+          mountPath: /etc/binderhub/custom
+  extraVolumes:
+    - name: custom-templates
+      emptyDir: {}
+  extraVolumeMounts:
+    - name: custom-templates
+      mountPath: /etc/binderhub/custom
+  dind:
+    enabled: true
+    daemonset:
+      image:
+        name: docker
+        tag: 19.03.5-dind
```

Comment:
We can just inherit the upstream binderhub default here, which is much more recent.

Reply:
Done in [`0ffba14` (#1404)](https://github.com/2i2c-org/infrastructure/pull/1404/commits/0ffba14f2411f4d5e756a675bf8bffa0c1147e66)